src/scheduler.c

```diff
@@ -129,18 +134,42 @@ static int avl_comp_timer(const void *entry1, const void *entry2) {
 uint32_t
 olsr_times(void)
 {
+#ifdef __APPLE__
+  uint32_t msec;
```

Comment:
I'd rather you keep an uint64_t here, so that we can keep a single calculation. just do a msec-->nsec multiplication